### DISCUSSION TITLE
How to add custom properties, such as tag or class

### DISCUSSION THREAD

**Discussion by hui3215** [vvvv/SVG#1112] [Q&A]
How to add custom properties, such as tag or class

**Comment by mrbean-bremen** [comment]
Do you mean custom attributes, as mentioned in #481?

**Comment by hui3215** [comment]
> Do you mean custom attributes, as mentioned in #481?

Thanks, I solved this problem, by the way ask the existing svg node string such as how can a rect be added to the svg file?
